oauth2client/dictionary_storage.py

```diff
@@ -0,0 +1,78 @@
+# Copyright 2014 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Dictionary storage for OAuth2 Credentials."""
+
+from oauth2client.client import OAuth2Credentials
+from oauth2client.locked_storage import LockedStorage
+
+
+__author__ = 'jonwayne@google.com (Jon Wayne Parrott)'
+
+
+class DictionaryStorage(LockedStorage):
```

Comment:
Drop `LockedStorage` from the inheritance hierarchy; I think your code will be merely a few tokens longer if `LockedStorage` is something that `DictionaryStorage` *uses* rather than something that it *is*.

As currently drafted this inheritance relationship violates the Liskov Substitution Principle because in requiring that a `DictionaryStorage` is satisfactory in every place that a `LockedStorage` is requested, it would first have to be the case that there are places where a `LockedStorage` is requested. But there are no such places, because the locking and unlocking is just a portion of `DictionaryStorage`'s implementation, not a part of itself that it really wants to export as a widely useful type.